### PR TITLE
Frontpage/projects

### PR DESCRIPTION
**Pull request by shreyaag770**
Rearranged the alignment under Project Section on Frontpage.

q1

### CODE REVIEW

**Comment by shreyaag770** [comment]
Alright! @oscerd I just thought this was a necessary change since @zregvart told us we can do minor changes other than issues listed. What should I do to this PR?

**Comment by oscerd** [comment]
I don't know, if this is a minor change I think you can go ahead without an issue number, when the updates are much bigger we need an issue for them.

**Comment by shreyaag770** [comment]
Thanks, @oscerd and Do you think changes I suggest in this PR are relevant? or else I can look for an issue in JIRA.

Changes I made :

Before:
q2

After:
q1

File: antora-ui-camel/src/css/frontpage.css [hunk 1]
```diff
@@ -50,24 +50,28 @@ header.frontpage a:active {
 
 .frontpage a.significant {
   margin: 1rem;
-  padding: 0.8rem 1.5rem;
-  background: var(--color-camel-orange);
-  color: var(--color-white);
+  padding: .8rem 1.5rem;
+  background: #ed8225;
+  color: #fff;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: We need to use the variables here.

File: antora-ui-camel/src/css/frontpage.css [hunk 2]
```diff
@@ -50,24 +50,28 @@ header.frontpage a:active {
...
   border-radius: 8px;
   font-size: 1.2rem;
-  font-weight: bold;
+  font-weight: 700;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Use weight names or variables instead of numbers.

File: antora-ui-camel/src/css/frontpage.css [hunk 3]
```diff
@@ -50,24 +50,28 @@ header.frontpage a:active {
...
   outline: none;
   display: inline-block;
   white-space: nowrap;
+  box-shadow: 0 4px #8e480b;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Buttons are designed without shadows, rebase and have a look at the current master or live website.

File: antora-ui-camel/src/css/frontpage.css [hunk 4]
```diff
@@ -156,8 +160,12 @@ section.frontpage.projects .project {
 
 section.frontpage.projects .project h1 {
   white-space: nowrap;
-  padding: 1rem 0;
-  flex: 1;
+  padding: 1rem ;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Trailing whitespace

File: antora-ui-camel/src/css/frontpage.css [hunk 5]
```diff
@@ -156,8 +160,12 @@ section.frontpage.projects .project {
...
+  -webkit-box-flex: 1;
+  -ms-flex: 1;
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: There is no reason to use browser prefixes.

File: antora-ui-camel/src/css/frontpage.css [hunk 6]
```diff
@@ -245,6 +253,10 @@ section.frontpage h1 {
     min-width: 51vw;
   }
 
+  section.frontpage.projects .project h1{
```
Comment (zregvart) [inline antora-ui-camel/src/css/frontpage.css]: Insert space after h1 and before the bracket

File: antora-ui-camel/src/css/frontpage.css (hunk 3 above)
Comment (shreyaag770) [reply antora-ui-camel/src/css/frontpage.css]: I have pulled the fresh code now and checked the current website, there have been updates, current buttons have a shadow.

**Comment by shreyaag770** [comment]
@zregvart I don't why but I am not able to push changes in this PR. I have to make another PR #264. All the changes you asked for are made in that PR. I am really sorry for all this trouble.

**Comment by zregvart** [comment]
No problem, lets close this in favour of #264 and continue the discussion there.

**Pull request closed without merging**